Extension/src/LanguageServer/extension.ts

```diff
@@ -1378,3 +1381,88 @@ export async function getIncludes(maxDepth: number): Promise<any> {
     const includes = await clients.ActiveClient.getIncludes(maxDepth);
     return includes;
 }
+
+// This uses several workarounds for interacting with the hover feature.
+// A proposal for dynamic hover content would help, such as the one here (https://github.com/microsoft/vscode/issues/195394)
+async function onCopilotHover(): Promise<void> {
+    if (!vscode.window.activeTextEditor) { return; }
```

Comment:
Hover is supposed to work on a document that is not the active editor.

Reply:
I see the hover disappear when Generate Copilot summary is clicked and there is no summary result.